openbas-front/src/actions/Inject.js

```diff
@@ -55,6 +55,11 @@ export const deleteInjectForExercise = (exerciseId, injectId) => (dispatch) => {
   return delReferential(uri, 'injects', injectId)(dispatch);
 };
 
+export const bulkDeleteInjectsForExercise = (exerciseId, injectIds) => (dispatch) => {
+  const uri = `/api/exercises/${exerciseId}/injects/delete`;
```

Comment:
I'm wondering if we shouldn't have a file with all URI ?

Reply:
open-api? /api-docs

Reply:
No, I meant having a file in the frontend with all the URI for API calls as constant so they are all in the same place.

Reply:
I don't think we have a file with all URLs, but we usually have a file for each resource.